spring-security-modules/spring-security-web-boot-4/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-security-web-boot-4</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>spring-security-web-boot-4</name>
+    <packaging>jar</packaging>
+    <description>Spring Security MVC Boot - 3</description>
```

Comment:
description should  be "MVC Boot - 4", no?

Reply:
yes thanks for spotting that